CONTRIBUTING.md

```diff
@@ -10,10 +10,13 @@ If you'd like to contribute, try contributing new symbols or functions that
 KaTeX doesn't currently support. The wiki has a page which lists [all of the
 supported
 functions](https://github.com/Khan/KaTeX/wiki/Function-Support-in-KaTeX) as 
-well as a page that describes how to [examine TeX commands and where to find
-rules](https://github.com/Khan/KaTeX/wiki/Examining-TeX) which can be quite
-useful when adding new commands. You
-can check there to see if we don't support a function you like, or try your
+well as a page that describes how to [examine TeX commands and where to find 
+rules](https://github.com/Khan/KaTeX/wiki/Examining-TeX) which can be quite 
+useful when adding new commands. There's also a user-contributed [preview page]
+(http://utensil-site.github.io/available-in-katex/)
+showing how would KaTex render a series of symbols/functions(including the ones
+MathJax listed in their documentation and the extra ones supported by KaTeX). You
+can check them to see if we don't support a function you like, or try your
```

Comment:
This is the original wording, unchanged. I guess it means that the contributor would look for what's missing and add them. Should I change the meaning to the other way around(the users looking for what's already at hand)?